pipelines/ocw/ocw-to-hugo.yml

```diff
@@ -7,29 +7,8 @@ jobs:
     trigger: true
   - get: ocw-json
     trigger: true
-  - task: sync-static-resources-task
-    config:
-      platform: linux
-      image_resource:
-        name: ''
-        source:
-          repository: amazon/aws-cli
-          tag: latest
-        type: docker-image
-      run:
-        path: sh
-        args:
-        - -exc
-        - >-
-          aws s3 sync
-          s3://((open-learning-course-data-bucket))/
-          s3://((ol-ocw-studio-app-bucket))/courses/
-          --include '*'
-          --exclude "extracts/*"
-          --exclude "*.html"
-          --exclude "*.html.json"
-          --exclude "*_master.json"
-          --exclude "*parsed.json"
+  - get: sync-static-resources
+    trigger: true
```

Comment:
This should be a `put` step so that it can take advantage of the bucket-to-bucket sync.

Reply:
Good call; and I will assume we do not need to `get` the `source_bucket` prior to the `put`.